### PR TITLE
Usage alert fix

### PR DESCRIPTION
**Pull request by NikitaPoly**

### CODE REVIEW

File: frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js [hunk 1]
```diff
@@ -190,7 +192,7 @@ export default function InteractiveWaveformModal({
     // only available for versions greater than 0.25.2
     const compatibleVersions = pulse3dVersions.filter((v) => semverGte(v, "0.25.2"));
     setFilteredVersions([...compatibleVersions]);
-    if (numberOfJobsInUpload > 2) {
+    if (usageQuota && usageQuota.limits && numberOfJobsInUpload > 2 && usageQuota.limits.jobs !== -1) {
```
Comment (luciipak) [inline frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js:195]: Is there always going to be `usageQuota.limits.jobs` if `usageQuota.limits` exists?

**Comment by luciipak** [comment]
I'd bump the patch version and upload to S3

File: frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js (hunk 1 above)
Comment (luciipak) [inline frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js:195]: Will `usageQuota.limits.jobs` always exist if `usageQuota` exists?

**Comment by NikitaPoly** [comment]
> 

I have already uploaded this to s3 and pointed the test cluster to this version before submitting the PR. I forget to bump the json version @luciipak

**Comment by luciipak** [comment]
0.3.7 is already being used to CloudFront would have already been pointing to v0.3.7, probably need to pull down latest changes from RC-02-16-23

**Comment by NikitaPoly** [comment]
> 0.3.7 is already being used so CloudFront would have already been pointing to v0.3.7, probably need to pull down latest changes from RC-02-16-23

Sorry I added v0.3.8 in aws and put 0.3.7 in the json. @luciipak . So I just need to update the commit.

**Review by luciipak — APPROVED** [review]

**Pull request merged**